docs/src/refs.bib

```diff
@@ -26,6 +279,119 @@ @Article{Kem21
   doi           = {10.2307/1989020}
 }
 
+@Article{LS78,
+  author        = {Lascoux, Alain and Sch{\"u}tzenberger, Marcel-Paul},
+  title         = {Sur une conjecture de {H}. {O}. {Foulkes}},
+  zbl           = {0374.20010},
+  journal       = {C. R. Acad. Sci., Paris, S{\'e}r. A},
+  fjournal      = {Comptes Rendus Hebdomadaires des S{\'e}ances de l'Acad{\'e}mie des Sciences, S{\'e}rie A},
+  volume        = {286},
+  pages         = {323--324},
+  year          = {1978},
+  language      = {French},
+  zbmath        = {3582368}
+}
+
+@Article{LS90,
+  author        = {Lambe, Larry and Srinivasan, Bhama},
+  title         = {A computation of {Green} functions for some classical groups},
+  zbl           = {0770.20021},
+  journal       = {Commun. Algebra},
+  fjournal      = {Communications in Algebra},
+  volume        = {18},
+  number        = {10},
+  pages         = {3507--3545},
+  year          = {1990},
+  doi           = {10.1080/00927879008824089},
+  language      = {English},
+  zbmath        = {5787}
+}
+
+@Article{Loo77,
+  author        = {Looker, John},
+  title         = {The complex irreducible characters of {Sp}(6, q), q even},
+  zbl           = {0367.20004},
+  journal       = {Bull. Aust. Math. Soc.},
+  fjournal      = {Bulletin of the Australian Mathematical Society},
+  volume        = {17},
+  pages         = {475--476},
+  year          = {1977},
+  doi           = {10.1017/S0004972700010741},
+  language      = {English},
+  zbmath        = {3572390}
+}
+
+@Book{Lub93,
+  author        = {L{\"u}beck, Frank},
+  title         = {Charaktertafeln f{\"u}r die {Gruppen} $\text{CSp}_6(q)$ mit ungeradem $q$ und $\text{Sp}_6(q)$ mit
+                  geradem $q$.},
+  zbl           = {0832.20022},
+  publisher     = {Heidelberg: Universit{\"a}t Heidelberg, Naturw.-Math. Gesamtfakult{\"a}t},
+  year          = {1993},
+  language      = {German},
+  zbmath        = {444262}
```

Comment:
Add link to https://www.math.rwth-aachen.de/~Frank.Luebeck/preprints/ or even directly to https://www.math.rwth-aachen.de/~Frank.Luebeck/preprints/CharTabCSp6.pdf ? Via the url field